src/cryptonote_basic/CMakeLists.txt

```diff
@@ -33,6 +33,7 @@ if(APPLE)
 endif()
 
 set(cryptonote_basic_sources
+  ../version.cpp
```

Comment:
This breaks linking for blockchain-import/export as they're not linked against cryptonote_basic.